.github/SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,127 @@
+# Security Insights 2.0 file https://github.com/ossf/security-insights 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)